### ISSUE TITLE
Make view service markers consistent for type explorer and repository explorer

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#3720]
Currently we have 'type-explorer' for tex, and 'rex' for rex. 
This is defined in ViewServiceDescription.java

ie
```
REPOSITORY_EXPLORER(1020,
                  "Repository Explorer",
                  "Repository Explorer OMVS",
                  "rex",
                  "Explore open metadata instances.",
                  //TODO create appropriate wiki
                  "https://odpi.github.io/egeria/open-metadata-implementation/access-services/subject-area/"),

    TYPE_EXPLORER(1030,
                  "Type Explorer",
                  "Type Explorer OMVS",
                  "type-explorer",
                  "Explore the open metadata types.",
                  //TODO create appropriate wiki
                  "https://odpi.github.io/egeria/open-metadata-implementation/access-services/subject-area/"),
```
They can be made more consistent

**Comment by mandy-chessell** [comment]
This is fixed in #3707 ...
This is the values that will be set to when the PR merges.

```
public enum ViewServiceDescription implements Serializable
{
    GLOSSARY_AUTHOR(5000,
                    "Glossary Author",
                    "Glossary Author OMVS",
                    "glossary-author",
                    "View Service for glossary authoring.",
                    "https://egeria.odpi.org/open-metadata-implementation/view-services/glossary-author-view/"),

    REPOSITORY_EXPLORER(5001,
                  "Repository Explorer",
                  "Repository Explorer OMVS",
                  "rex",
                  "Explore open metadata instances.",
                  "https://egeria.odpi.org/open-metadata-implementation/view-services/rex-view/"),

    TYPE_EXPLORER(5002,
                  "Type Explorer",
                  "Type Explorer OMVS",
                  "type-explorer",
                  "Explore the open metadata types.",
                  "https://egeria.odpi.org/open-metadata-implementation/view-services/tex-view/"),

    DINO(5003,
                  "Dino",
                  "Dino OMVS",
                  "dino",
                  "Explore and operate an open metadata ecosystem.",
                  "https://egeria.odpi.org/open-metadata-implementation/view-services/dino-view/");

```

**Comment by grahamwallis** [comment]
The above doesn't address the inconsistency... it's the 4th field that is inconsistent  - for RepositoryExplorer it is "rex" and for TypeExplorer it is "type-explorer". They should either both be long or both short.

**Comment by mandy-chessell** [comment]
Fair enough :)

**Comment by grahamwallis** [comment]
Fixed by #3736
